### ISSUE TITLE
Invalid Username Password

### ISSUE THREAD

**Issue by goku-son** [tidusjar/Ombi#2136]
#### Ombi build Version:

V 3  Latest

#### Update Branch:

Stable

#### Media Sever:

Plex

#### Media Server Version:

1.12

#### Operating System:
Windows 10

#### Ombi Applicable Logs (from `/logs/` directory or the Admin page):

```
No errors in log :(

```

#### Problem Description:

Trying to login and the page seems to reset itself and tells me I supplied a bad user name or password. I get this for any user I try (no password required) or the plex admin user. If I select that I forgot my password it sends me and email that redirects me to the plex.tv so I can reset my plex password which works so I don't want to do that. Any suggestions short of blowing away the db and starting over? This has been working just fine for 2 weeks. 

#### Reproduction Steps####
Try and log in.

**Comment by tidusjar** [comment]
So do you have require password enabled?

Can local accounts login?
Is it only Plex accounts?

What's in the browsers console when trying to log in, any errors?

**Comment by pgkahnk** [comment]
I am having a similar issue though my 'invalid' password only presents itself after a reboot of the host OS (Centos7 in my case).  I am running ( Centos7 > docker > linuxserver/ombi:latest ).  I get the invalid password on my Plex account as well as locally created accounts in Ombi.  I have reproduced this a couple times trying to see what I am doing wrong.  When it locks up I just delete the Ombi.db file and restart the container.  I have a feeling it is caused with not properly closing the Ombi.db before a host reboot?  Just guessing at this point though.

**Comment by tidusjar** [comment]
Hmmm. Well if the dB is locked then yes we won't be able to read any user. Is someone able to post logs or the browser errors?

**Comment by goku-son** [comment]
Hi @tidusjar I think this was related to some dns issues I was having. After addressing those problems, I am unable to reproduce the problem. The strange thing for me at least was I did not get any errors in the log and on the GUI, it would seem to be stuck in some sort of loop where it refused to recognize any local home plex users nor my main plex account. No auth for users is enabled and my plex admin account uses auth being a ombi admin account. Any user I tried would result in "Invalid User/Password " type of message (sorry can't recall the exact verbiage of the message. 
My DNS servers were being autoblocked by my firewall if that helps anyone that may be encountering this problem.

**Comment by goku-son** [comment]
Perhaps I spoke too soon. I am seeing the issue again and not having any DNS issues. So here is what I see which is bizarre:

1. Supply user name and click submit 
2. Immediately the background changes to one of my movies or shows
3. I then get a black ombi background
4. Then I get another tv or movie loaded as the background 
5. Finally, I get the Invalid User ID or Password error

It almost seems like the background image on the login page causes the user to be incorrectly recognized some how. Not sure this is what the issue but that is the behavior I observe when I see this issue.

**Comment by goku-son** [comment]
Totally think it is related to the background image. When that background image in the background does not load, I am able to login. When that background starts loading, I get the above behavior.

**Comment by tidusjar** [comment]
Can someone please post any errors in the browsers console when this happens? Press F12 then Console.

**Comment by pgkahnk** [comment]
I checked the server and ntp was in sync.  This is interesting though, I rebooted the centos vm this morning, after it booted and ombi was back up and running for a couple minutes I tried to login and the username auth failed.  I left it at that and went to work.  I just checked (12+hrs later) and I am able to login now (no rebootes, no fresh installs, it just sat all day).  Does the auth fail if it hasn't pulled all the information from the plex servers yet, or plex doesn't respond?  When I rebooted the vm this morning, the ombi instance was running for a couple minutes before I tried to login.  I would have thought that would have been enough time for everything to initialize.  Another note, Ombi also seems to have me sign in multiple times (every couple minutes) while I am going through the settings and user management areas.

**Comment by goku-son** [comment]
Have not be able to replicate this issue again, but will keep and eye out and collect the browser console output should it happen again.

**Comment by anojht** [comment]
Can you guys update to the latest release and test to see if you have the same issue. Please press F12 if you are using chrome, right click the refresh button and choose "Empty cache and hard reload" option before testing. 

If the issue is no longer present please close this issue, thanks!

**Comment by pgkahnk** [comment]
So I still have the invalid password issue on the latest release, but only if I don't check the 'remember me' box before signing in.  If I check that box before clicking 'sign in' I am logged in just fine.  I don't recall if I ever checked that box on prior tests.

**Comment by anojht** [comment]
This has to do with time being out of sync please see #2048 as I stated earlier.

**Comment by pgkahnk** [comment]
I looked over the Plex server VM, the Ombi container, and the docker host.  It turned out that the Plex server was set for UTC and the Docker host and container was set for CDT (while the time was in sync between all three, the offset from universal was different for the Plex server).  It still confuses me why local accounts still failed when the authentication should not have gone through Plex.

Since this seems to match exactly as #2048 and I can't get it to fail after I set all offsets the same.  This needs to be closed.   Sorry for the hassles.  As always, great work with the tool.

**Comment by ziptbm** [comment]
Is there another workaround to this?  I see this regularly on the latest build, but my time is in sync across all components.  Password is definitely correct, however I essentially get locked out and have to rebuild the entire app.

**Comment by chrislolds** [comment]
I am also having this issue.

New Docker Install on setup screen, won't get auth directly from plex so I click on login with plex, that works and gets me into ombi.   Check username in uper right hand corner. Log out.
Immediately try to log back in and can't, wrong password.
Checked time on Ombi Docker, Plex Docker and unraid host, all set to local time zone and correct.

Any ideas?

**Comment by myrison** [comment]
I believe I'm having the same issue, but it could be I don't understand how user authentication is supposed to work either as I'm a brand new user...

My users are plex users, I have Plex OAuth disabled in Settings / Configuration / Authentication (which I assume means they login with a password that I create for them as a new user, or that they correct after clicking reset password). Is this correct, or is it linked to Plex regardless?

If it is meant to be a separate password, then I'm not able to assign a new password to a user account and have it work.  I tried resetting a number of passwords and always get the invalid login.  The only thing in the logs is this:

2019-12-22 15:33:40.310 -07:00 [Error] StatusCode: Unauthorized, Reason: Unauthorized, RequestUri: https://plex.tv/users/sign_in.json

For now I've turned off the password requirement and things are working, but I'd prefer to have a password if possible.  Thanks for the wonderful software!
